WordPress/src/main/java/org/wordpress/android/ui/uploads/PostUploadNotifier.java

```diff
@@ -419,7 +420,8 @@ public static long getNotificationIdForMedia(SiteModel site) {
         }
     }
 
-    void updateNotificationErrorForPost(@NonNull PostModel post, @NonNull SiteModel site, String errorMessage) {
+    void updateNotificationErrorForPost(@NonNull PostModel post, @NonNull SiteModel site, String errorMessage,
```

Comment:
Can we add a comment here explaining what `overrideMediaNotUploadedCount` is? I think if we have this explainer, we can drop the `K_DONT_OVERRIDE_MEDIA_COUNT` constant and simply pass `0`.

Reply:
ok, addressed in 284d681, would you check whether the comment seems clear enough please? thx

Reply:
Looks good, except for the minor typo ("youn" instead of "you").

Reply:
😁 545b4a1